src/api/ApiAccessors.py

```
@@ -174,12 +164,12 @@ def parseSingleData(self, data: ApiResourceObject, included: dict[str, Any]) ->
                     result[key] = value
             if "relationships" in data:
                 for key, value in data["relationships"].items():
-                    result[key] = self.parseData(value, included)
+                    result[key] = self.parse_data(value, included)
         except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)